setup.py

```diff
@@ -75,22 +75,22 @@
     url="https://github.com/ethereum/web3.py",
     include_package_data=True,
     install_requires=[
-        "aiohttp>=3.7.4.post0,<4",
-        "eth-abi>=3.0.0,<4.0.0",
-        "eth-account>=0.7.0,<0.8.0",
-        "eth-hash[pycryptodome]>=0.2.0,<1.0.0",
-        "eth-typing>=3.0.0,<4.0.0",
-        "eth-utils>=2.0.0,<3.0.0",
-        "hexbytes>=0.1.0,<1.0.0",
+        "aiohttp>=3.7.4.post0",
+        "eth-abi>=3.0.0",
```

Comment:
We can probably stick this to `3.0.1` already maybe?

Reply:
Since `web3.py` doesn't use any new features in `eth-abi 3.0.1` I think it might be better to leave it as open as possible.

Reply:
sgtm 👍🏼